### PR TITLE
Updated docker related links and added link to tutorial in README

### PR DESCRIPTION
**Pull request by tiagofilipe12**
Performed the updates described in the title.

### CODE REVIEW

**Comment by tiagofilipe12** [comment]
Added tutorial and blog entry about  work-flow managers to main README.md. You can now check it and see if everything is ok and if so merge it. However, note that issue [49](https://github.com/bionode/bionode-watermill/issues/49) should also address the creation of tasks with js functions and streams.

**Comment by tiagofilipe12** [comment]
Added a way to create a manifest file that allow to make a correspondence between folders created within `data` and the task names.

File: lib/sagas/lifecycle.js [hunk 1]
```diff
@@ -108,6 +114,27 @@ function* lifecycle (action) {
   logEmitter.emit('log', `=== ${originalTask.name} ===`)
   logEmitter.emit('log', `Running task ${miniUid}`)
 
+  // First handles the possibility of input, output and params being an
+  // object rather than a string (e.g. when multiple params are passed to
+  // the task).
+  let string_output = JSON.stringify(originalTask.output)
```
Comment (thejmazz) [inline lib/sagas/lifecycle.js]: use camelCase

File: lib/sagas/lifecycle.js [hunk 2]
```diff
@@ -108,6 +114,27 @@ function* lifecycle (action) {
...
+  let string_input = JSON.stringify(originalTask.input)
+  let string_params = JSON.stringify(originalTask.params)
+  // Then creates the string to be written to the log file
+  let manifest_log =
+  `{ folder_name: "${originalTask.dir}",
```
Comment (thejmazz) [inline lib/sagas/lifecycle.js]: this should just be an object that you stringify

File: lib/sagas/lifecycle.js [hunk 3]
```diff
@@ -108,6 +114,27 @@ function* lifecycle (action) {
...
+  let string_params = JSON.stringify(originalTask.params)
+  // Then creates the string to be written to the log file
+  let manifest_log =
+  `{ folder_name: "${originalTask.dir}", 
+  task_name: "${originalTask.name}", 
+  input: ${string_input}, 
+  output: ${string_output}, 
+  params: ${string_params} }\n`
+  // Writes to the file
+  fs.appendFile('summary_log.txt', manifest_log, function (err) {
```
Comment (thejmazz) [inline lib/sagas/lifecycle.js]: maybe logfile should be ndjson (one json object per line)

**Comment by tiagofilipe12** [comment]
I have separated the tutorial updates from the summary log file commits, in order to simplify the revision. So, if you feel this is ok, please feel free to merge.

**Review by thejmazz — APPROVED** [review]

**Pull request merged**